tensorflow/tools/tf_windows_build_dockerfiles/Dockerfile_py3.9

```diff
@@ -0,0 +1,111 @@
+# This Dockerfile creates an image that:
```

Comment:
Since you have 2 files that should mostly be identical, any thoughts on removing duplication? We want to prevent edits to one file which are accidentally not propagated to the other one

Reply:
Agreed this should follow how linux has a single file for the different versions.

Reply:
Done